<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationWndClass</class>
 <widget class="QMainWindow" name="ApplicationWndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ApplicationWnd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="resultTable"/>
     </item>
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>Connect to Database</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="runBtn">
         <property name="text">
          <string>Run Query</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="queryEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actionBarcode"/>
   </widget>
   <addaction name="menuOutils"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSQL">
   <property name="text">
    <string>SQL</string>
   </property>
  </action>
  <action name="actionBarcode">
   <property name="text">
    <string>Barcode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ApplicationWnd.qrc"/>
 </resources>
 <connections/>
</ui>
